src/components/Component.css

```diff
@@ -1,6 +1,6 @@
 .center-button {
   position: absolute;
-  left: 48%;
+  left: calc(50% - 90px);

```

Comment:
why this change? did you test with different window sizes?

Reply:
essentially the button width is set to 180px; so it will always be centered with this calc